react/gulpfile.js

```diff
@@ -56,7 +185,8 @@ const sources = [
   '!src/**/*.knobs.options.js',
   '!src/**/*.knob.options.js',
   '!src/**/Colors/**',
-  '!src/**/storyutils.js'
+  '!src/**/storyutils.js',
```

Comment:
I don't see files named storyutils.js

Reply:
Good catch. Not sure why I added that...